README.md

```diff
@@ -149,6 +149,25 @@ documentation and rspec tests. We are happy to provide guidance if necessary.
 
 Please see [CONTRIBUTING](.github/CONTRIBUTING.md) for more details.
 
+### Multihost integration tests
+
+This repo includes some integration tests that require more than one host.
```

Comment:
we currently only manage https://github.com/voxpupuli/puppet-mongodb/blob/master/.github/workflows/ci.yml via modulesync. you could place a second file next to it that runs the multinode tests. I'm not sure if we can manipulate the role via any parameter at the moment :thinking:

Reply:
I'll see if I find some time to work on that in another PR.
I think the best thing here would be to add this to `metadata2gha`.
Something like `inputs.beaker_facter`

Reply:
Hmm if we could get `metadata2gha` to generate something like
```
    {
        "name": "Puppet 8 - Ubuntu 22.04",
        "setfile": {
            "name": "Ubuntu 22.04",
            "value": "ubuntu2204-64master,shard.-64slave,router."
        },
        "puppet": {
            "name": "Puppet 8",
            "value": 8,
            "collection": "puppet8"
        }
    },
```

Reply:
Solved by overriding modulesync generated ci.yml